pyproject.toml

```diff
@@ -210,3 +210,71 @@ markers = [
   "requires_docs: mark tests that can only be run with documentation present (deselect with '-m \"not requires_docs\"')",
   "requires_internet: mark tests that require internet access (deselect with '-m \"not requires_internet\"')"
 ]
+
+[tool.ruff]
+src = ["xclim"]
+line-length = 150
+target-version = "py38"
```

Comment:
why specify "target-version" here but this is dropped in  "nbqa-black"? Just not sure when this is needed or not

Reply:
I'm not entirely sure what the target-version here impacts, but it should affect any hooks from Ruff that require a minimum supported Python version.

The configuration options for black now live in the `pyproject.toml` (and have been for a few months now). If the CLI flags for black are not set manually, `nbqa-black` will use that configuration by default.